Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Meter/ByMeter.tsx

```diff
@@ -160,7 +160,7 @@ const ByMeter: Application.Types.iByComponent = (props) => {
                             padding: 0, width: '100%', height: '100%',
                             tableLayout: 'fixed', overflow: 'hidden', display: 'flex', flexDirection: 'column', marginBottom: 0
                         }}
-                        TheadStyle={{ fontSize: 'auto', tableLayout: 'fixed', display: 'table', width: '100%' }}
+                        TheadStyle={{ fontSize: 'smaller', tableLayout: 'fixed', display: 'table', width: '100%' }}
```

Comment:
```suggestion
```
Comparing this to default in `gpa-gemstone`  you can drop this entirely.

```
const defaultHeadStyle: React.CSSProperties = {
    fontSize: 'auto', 
    tableLayout: 'fixed', 
    display: 'table', 
    width: '100%'
};
```

Reply:
Good catch, thank you